ocaml/typing/typemod.ml

```diff
@@ -3522,7 +3522,7 @@ let cms_register_toplevel_attributes ~sourcefile ~uid ~f ast =
   match attr with
   | None -> ()
   | Some attr ->
-    Cms_format.register_toplevel_attributes uid
+    Cms_format.register_toplevel_attributes (uid : Uid.t)
```

Comment:
This change looks like a debugging fragment. If it is, please remove.

Reply:
Yes, good catch, thanks

Reply:
Removed